code/modules/shuttle/shuttle.dm

```diff
@@ -247,16 +276,24 @@
 	if(height-dheight > S.height-S.dheight)
 		return 5
```

Comment:
Not sure if this is exactly within scope for your PR, but this looks like it should be #defines instead of magic numbers